### PR TITLE
Avoid jar cross-contamination by doing 3 separate buildRelease invocations

### PR DESCRIPTION
**Pull request by raboof**
Refs #28807

### CODE REVIEW

**Comment by raboof** [comment]
I have only tested this by invoking `buildRelease` locally, without actually publishing to maven central.

Perhaps the 2.5.31 release should be the real test?

File: project/scripts/release [hunk 1]
```diff
@@ -298,10 +298,11 @@ if [ ! $dry_run ]; then
...
 fi
-try sbt $RELEASE_OPT +buildRelease
-try sbt -Dakka.build.scalaVersion=2.11.12 -D$RELEASE_OPT buildRelease
-try sbt $RELEASE_OPT clean
-try sbt $RELEASE_OPT buildDocs
+# Needs to be kept in sync with project/Dependencies.scala manually
+try sbt -Dakka.build.scalaVersion=2.12.8 $RELEASE_OPT clean buildRelease
+try sbt -Dakka.build.scalaVersion=2.13.0 $RELEASE_OPT clean buildRelease
+try sbt -Dakka.build.scalaVersion=2.11.12 $RELEASE_OPT clean buildRelease
+try sbt $RELEASE_OPT clean buildDocs
```
Comment (patriknw) [inline project/scripts/release]: just to cut down on the build time we could maybe build 2.12 last and then not clean before buildDocs? (but not important)

**Comment by patriknw** [comment]
We should also change the jenkins snapshot publishing https://jenkins.akka.io:8498/job/akka-publish-2.5-nightly/

**Comment by raboof** [comment]
> We should also change the jenkins snapshot publishing https://jenkins.akka.io:8498/job/akka-publish-2.5-nightly/

done

**Review by patriknw — APPROVED** [review]
LGTM

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**